compiler/circle-opselector-test/README.md

```diff
@@ -0,0 +1,52 @@
+# Circle-OpSelector-test
+
+## Usage
+
+- This test didn't consider **if** and **while** nodes yet. 
+
+**Single file test**
+
+- `tflite-file-name` must be contained in [tflite receipes](https://github.com/Samsung/ONE/tree/master/res/TensorFlowLiteRecipes)
+- `selected nodes` format is `"0 1 3"`
+
+```
+$ ./opselector_test.sh [tflite-file-name] [selected nodes]
+# ex)
+$ ./compiler/circle-opselector-test/opselector_test.sh Part_Sqrt_Rsqrt_Add_002 "0 1 2 3"
+```
+
+This test compares between the result of `circle-opselector` and [select-operator.py](https://github.com/Samsung/ONE/blob/master/tools/tflitefile_tool/select_operator.py) in tflite_tool.
+
+The detailed process is as follows
+
+1. Prepare a `tflite` model.
+2. Create a `circle` version of that `tflite` model using 'tflite2circle`.
+3. Create a `tflite` subgraph using `select_operator.py`.
+4. Create a `circle` subgraph using `Circle-OpSelector`.
+5. Run `nnpckgtc` with input as the `tflite` subgraph to get the golden value and the package.
```

Comment:
`nnpckgtc` ?

Reply:
`nnpckgtc` module can convert tflite file to nnpackage and generate input and output data. So if output value of selected circle model and output golden value are same, then selected circle model will be correct

Reply:
Isn't `nnpckgtc` a typo ?

Reply:
Oh it is a typo. `nnpkgtc` is correct